### PR TITLE
Add extra conversions

### PR DESCRIPTION
**Pull request by tomhoule**
- Postgres BYTEA
- SQLite datetime text format
- deserialize_bytes in serde module

### CODE REVIEW

**Comment by tomhoule** [comment]
I think this (sqlite and postgres bytea) should be tested, but we (@do4gr and me) don't have a lot of bandwidth at the moment. I'll try to find time.

**Comment by tomhoule** [comment]
I'll probably drop the BYTEA work (we don't need it immediately anymore, and too much hurry to spend time on testing it) and just keep the SQLite datetime FromSql implementation

**Comment by pimeys** [comment]
Tests are commented out, just put them back and see if you get them green. https://github.com/prisma/quaint/blob/master/src/tests/types/postgres.rs#L365

**Comment by tomhoule** [comment]
@pimeys the postgres bytea test. Can't figure out how to test the sqlite conversion in the roundtrip tests. It's necessary for the `serde` feature to work with datetimes on SQLite.

File: src/connector/postgres/conversion.rs [hunk 1]
```diff
@@ -116,6 +116,13 @@ impl GetRow for PostgresRow {
                     }
                     None => Value::Real(None),
                 },
+                PostgresType::BYTEA => match row.try_get(i)? {
+                    Some(val) => {
+                        let val: &[u8] = val;
```
Comment (yoshuawuyts) [inline src/connector/postgres/conversion.rs:121]: Is this line needed? If it's for inference of `try_get`, maybe use turbofish there instead?

Comment (pimeys) [reply src/connector/postgres/conversion.rs:121]: yeah it's the `try_get` above: https://docs.rs/tokio-postgres/0.5.5/tokio_postgres/row/struct.Row.html#method.try_get

Comment (tomhoule) [reply src/connector/postgres/conversion.rs:121]: It's that way to align with the other branches. I think the tokio-postgres doc show something like that as well. Agreed I'd prefer a turbofish.

**Pull request merged**